site/package.json

```diff
@@ -13,7 +13,7 @@
 		"dev": "vite",
 		"format": "biome format --write .",
 		"format:check": "biome format .",
-		"lint": "pnpm run lint:check && pnpm run lint:types && pnpm run lint:circular-deps",
+		"lint": "pnpm run lint:check && pnpm run lint:types && pnpm run lint:circular-deps && knip",
```

Comment:
we should add this as it's own `lint:knip` script and invoke that just so it can be run independently

Reply:
I originally had that but running `pnpm knip` already runs knip so it seemed a little redundant